tools/topology/topology2/avs-tplg/tplg-targets.cmake

```diff
@@ -1,18 +1,18 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
 # Array of "input-file-name;output-file-name;comma separated pre-processor variables"
-set(TPLGS
+list(APPEND TPLGS
```

Comment:
This change is not related to topic, unless it is part of the 'preparation' umbrella.

Reply:
Correct, this could  have been another "preparation" commit. However I felt like this commit was small and easy enough to review. Moreover this CMake change makes no output difference at this early stage.